std/io/readers_test.ts

```diff
@@ -36,3 +37,15 @@ test("ioMultiReader", async function (): Promise<void> {
   await copy(r, w);
   assertEquals(w.toString(), "abcdef");
 });
+
+test("ioLimitedReader", async function (): Promise<void> {
+  let r = new LimitedReader(new StringReader("abc"), 2);
+  let buffer = await Deno.readAll(r);
+  assertEquals(decode(buffer), "ab");
+  r = new LimitedReader(new StringReader("abc"), 3);
+  buffer = await Deno.readAll(r);
+  assertEquals(decode(buffer), "abc");
+  r = new LimitedReader(new StringReader("abc"), 4);
+  buffer = await Deno.readAll(r);
+  assertEquals(decode(buffer), "abc");
+});
```

Comment:
Consider adding this test as well: https://github.com/golang/go/blob/f8662a5a96698c5fcf3a77f692e425c7b857e681/src/io/io_test.go#L35-L48

Reply:
Added